scripts/HPC_chatbot_preprocessor/parsed_mds/generic/FAQ/FAQ_paragraph_5.txt

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash 
+echo "hello world"
+And we run it like mympirun ./hello.sh --output output.txt.
+To our surprise, this doesn't output to the file output.txt, but to
+standard out! This is because mympirun expects the program name and
+the arguments of the program to be its last arguments. Here, the
+--output output.txt arguments are passed to ./hello.sh instead of to
+mympirun. The correct way to run it is:
+mympirun --output output.txt ./hello.sh
+When will my job start?
+See the explanation about how jobs get prioritized in When will my job start. 
```

Comment:
this is annoying, looks like a self referring paragraph. maybe add a short summary here and point ot the other url for full details